ldclient/twisted_sse.py

```diff
@@ -0,0 +1,170 @@
+from __future__ import absolute_import
+
+from copy import deepcopy
+from ldclient import log
+from twisted.internet.defer import Deferred
+from twisted.internet.ssl import ClientContextFactory
+from twisted.web.client import Agent
+from twisted.web.http_headers import Headers
+from twisted.protocols.basic import LineReceiver
+
+
+class NoValidationContextFactory(ClientContextFactory):
+    def getContext(self, *_):
+        return ClientContextFactory.getContext(self)
+
+
+class TwistedSSEClient(object):
+    def __init__(self, url, headers, verify, on_event):
+        self.url = url
+        self.verify = verify
+        self.headers = headers
+        self.on_event = on_event
+        self.on_error_retry = 30
```

Comment:
Feels a bit long. I would think something like 2 seconds is more reasonable on a non-200 response to handle network blips.